app/docs/0.10.x/configuration.md

```diff
@@ -203,6 +204,38 @@ http {
 }
 ```
 
+#### Serving API and Website
+
+A common use case for this is to serve a website and API from the
+same server name and port. Having a custom configuration file with
+`nginx_kong.lua` inlined, you could add a new location block for serving
+the website:
+
+```
+# ---------------------
+# custom_nginx.template
+# ---------------------
+
+worker_processes ${{NGINX_WORKER_PROCESSES}}; # can be set by kong.conf
+daemon ${{NGINX_DAEMON}};                     # can be set by kong.conf
+
+pid pids/nginx.pid;                      # this setting is mandatory
+error_log logs/error.log ${{LOG_LEVEL}}; # can be set by kong.conf
+
+events {}
+
+http {
+  # inline Kong configuration template here
+  ...
+  location /website {
```

Comment:
The intent from API providers is usually to serve the website from root (`/`), and the API from some arbitrary path (`/api/v1`).

To better reflect the example, I would advise showing the inlined Kong configuration file (to some extent, only the relevant part, by leaving the boilerplate ngx_lua directives in `...` placeholders), and highlight how the Kong Proxy handler `location /` has been changed to something like `location /api/v1`. Then, we add a comment over this line, saying something like:

```nginx
# here, we declare our custom location serving our website (or API portal)
# which we can optimize for serving static assets
location /api/v1 {
    root /var/www/my-api.com/html;
    index index.htm index.html;
    ...
}
```